Documentation/admin/install-multi.md

```diff
@@ -0,0 +1,175 @@
+<br>
+<div class=“alert alert-info” role=“alert”>
+<i class=“fa fa-exclamation-triangle”></i><b> Note:</b> This documentation is for an alpha feature. For questions and feedback on the Multi-cluster Alpha program, email <a href="mailto:tectonic-alpha-feedback@coreos.com">tectonic-alpha-feedback@coreos.com</a>.
+</div>
+
+# Installing the multi-cluster registry
+
```

Comment:
Multi-cluster can be installed on track-1 clusters by applying the manifests from https://github.com/coreos-inc/tectonic-operators/tree/cca6cb2/operator/utility/manifests/directory-syncer. Not sure how to best present this to the user, @zbwright any thoughts?

Since the [Secret](https://github.com/coreos-inc/tectonic-operators/blob/cca6cb2/operator/utility/manifests/directory-syncer/directory-sync-secret.yaml) to the directory cluster is created [later in the doc](https://github.com/coreos/tectonic-docs/pull/118/files#diff-fd38c5e6f5792ebe67f7a4ecb6766f11R130) we shouldn't include it here or should move those instructions to this section.

Reply:
the tectonic-operators repo is private. we can, however, link to individual manifests contained within it, allowing users to download them. 

Give me the instructions for how to apply these manifests, and I will fold the linked download into them. 

We should also add a note that users may not be able to update any multi-cluster setup they create using this process in the future, right? (We can't talk about track-1 vs. track-2, but we should make it clear that this is an alpha product, and may become a dead end in the future.)

@robszumski ?

Reply:
> Give me the instructions for how to apply these manifests, and I will fold the linked download into them.

The manifests can be installed with `kubectl apply --record -f <url to manifest>`.

Reply:
also, most of these are short, I would almost simply copy them into the body of the doc. @robszumski ?

Reply:
@ethernetdan I think we tell users to apply directory-sync-secret.yaml somewhere within the ## Set up replica cluster(s) section - I believe it should be after uploading the kubeconfig, and before checking that it deployed correctly. I'm going to move it there, and ask you to confirm, please.

Reply:
I vote for copying them into the body of the doc. I think that's easier than trying to host them somewhere.